include/lsst/jointcal/PhotometryMapping.h

```diff
@@ -169,6 +207,19 @@ class ChipVisitPhotometryMapping : public PhotometryMappingBase {
                                                       tempFlux);
     }
 
+    /// @copydoc PhotometryMappingBase::transformError
+    double transformError(MeasuredStar const &measuredStar, double instFluxErr) const override {
+        double tempFluxErr = _chipMapping->transformError(measuredStar, instFluxErr);
+        return _visitMapping->getTransfoErrors()->transform(measuredStar.getXFocal(),
+                                                            measuredStar.getYFocal(), tempFluxErr);
+    }
```

Comment:
This is purely a stylistic comment you can feel free to ignore.

I wonder if it would be better to overload `transform` and `transformError` to also except `int &x`, `int &y`. This would allow you to use
```
_visitMapping->transform(measuredStar.getXFocal(), measuredStar.getYFocal(), tempFluxErr);
```
which seems less clunky to me.

Reply:
I'm confused about your `int&` suggestion: none of these methods take `int`-anything.

Did you mean that I should overload `PhotometryMapping.transform()` to take two `double &`? Mappings are very explicit in operating on MeasuredStars, and then taking the necessary position components internally.

Although your comment did get me to see a bug in the above, and also reminded me that I should actually write some unittests for `freezeErrorScales`. I'll do that and request a review of the changes.

Reply:
Yes, I was thinking that you could overload PhotometryMapping.transform() to take two doubles, so that you wouldn't have to have `getTransfoErrors()` to access the private member. But if there is already a different standard then it's fine.